server.js

```diff
@@ -270,39 +272,94 @@ app.use('/pl/course_instance/:course_instance_id/instructor/assessment/:assessme
 app.use(/^(\/pl\/course_instance\/[0-9]+\/instructor\/assessment\/[0-9]+)\/?$/, (req, res, _next) => {
     res.redirect(`${req.params[0]}/questions`);
 });
+app.use('/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id', [
+    function(req, res, next) {
```

Comment:
Can we format these navPage-setting functions onto a single line? Like:
```
function(req, res, next) {res.locals.navPage = ‘XXX’; next();},
```
I don’t object to putting them in the main `server.js`, but I worry about `server.js` getting too verbose and thus hard to scroll through when trying to understand the high-level app structure.

Reply:
done!